pyproject.toml

```diff
@@ -58,6 +58,7 @@ source = ["deptry"]
 
 [tool.mypy]
 files = ["deptry", "scripts", "tests"]
+explicit_package_bases = true
```

Comment:
`scripts` directory doesn't have an `__init__.py` file, so this makes `mypy` understand that `scripts` is a Python module (https://mypy.readthedocs.io/en/stable/config_file.html?highlight=explicit#confval-explicit_package_bases).